src/components/molecules/TodoItem.vue

```diff
@@ -13,30 +13,48 @@ const toggleTodo = (todo) => store.dispatch('toggleTodo', todo)
 </script>
 
 <template>
-  <div class="todo-item">
-    <AppCheckboxInput :checked="props.todo.completed" @change="toggleTodo(todo)" />
-    <div>
-      <span
-        :class="{
+  <a href="#" class="list-group-item d-flex align-items-center">
```

Comment:
Why it is a link? I guess it should be a div
```suggestion
  <div class="list-group-item d-flex align-items-center">
```